### ISSUE TITLE
Doesn't compile on rustc 1.13.0-nightly (3c5a0fa45 2016-08-22)

### ISSUE THREAD

**Issue by Eilie** [serde-rs/aster#102]
```
error: no associated item named `None` found for type `syntax::ast::FunctionRetTy` in the current scope
  --> /home/eilie/.cargo/registry/src/github.com-1ecc6299db9ec823/aster-0.22.1/src/fn_decl.rs:98:22
      |
98 |         let ret_ty = ast::FunctionRetTy::None(self.span);
      |  
```

**Comment by dtolnay** [comment]
You need to update your aster dependency. This was fixed in 0.25.0.

https://github.com/serde-rs/aster/blob/v0.25.0/src/fn_decl.rs#L102

**Comment by Eilie** [comment]
Thanks!
